lib/glific/flows/flow.ex

```diff
@@ -93,11 +95,17 @@ defmodule Glific.Flows.Flow do
     validate_keywords(changeset, get_change(changeset, :keywords))
   end
 
+  @spec check_for_empty_keyword?(map()) :: map()
+  defp check_for_empty_keyword?(attrs) do
+    attrs = %{keywords: []} |> Map.merge(attrs)
+    if attrs.keywords == nil, do: Map.merge(attrs, %{keywords: []}), else: attrs
```

Comment:
Why we need this nil check here? Since we are using this one on the validate_keywords already?